### PR TITLE
added output methods and verbosity flag

### LINKED ISSUE #37: File Read `Print` Statements Contaminate Output
When reading a file, the file reading channel outputs some basic status information. However, this causes an issue that, if the output from a file read is piped to another command, the printed lines will come along for the ride.

For example, if the file 'input' contains the string "testing" and the following command is run:

'./screep receive -c file -e identity -p '{"file": {"filename": "input"}}' | ./screep send -c file -e identity -p '{"file": {"filename":"output"}}'

Then the resulting output in the file "output" is:
Reading data from input...
Done.
testing

Obviously, this example is contrived, but Sneaky-Creeper has an unlooked for utility in that it can be used now to encode arbitrary data (for example by piping a plaintext file read through a file write with rsa encoding). We may want to resolve this issue, and keep future outputs clean so that we can extend sneaky-feature's toolset as a general encoder/decoder with a large variety of possible channels.

### PR DESCRIPTION
**Pull request by DakotaNelson**
To resolve #37.

Haven't removed print statements from channels/encoders yet.

### CODE REVIEW

File: screep [hunk 1]
```diff
@@ -38,6 +39,7 @@ parser_encoders = subparsers.add_parser('encoders')
 parser_channels = subparsers.add_parser('channels')
 #parser_channels.add_argument()
 
+
```
Comment (ManickYoj) [inline screep:42]: Daring.

**Comment by ManickYoj** [comment]
Hold off on merging this - it doesn't currently resolve #37 .

My test command:
`echo "testing" |./screep send -e identity -c file -p '{"file": {"filename": "test"}}'`

Creates a file that says `a b testing` instead of `testing`.

Furthermore, receiving that and sending it back to a file results in the same output as #37 , but now with "a b " prepended to it.

Test command:
`./screep receive -e identity -c file -p '{"file": {"filename": "test"}}' | ./screep send -e identity -c file -p '{"file": {"filename": "test"}}'`

Results in:
`a b Reading data from test...
Done.
testing`

**Comment by DakotaNelson** [comment]
The a b is due to packetization - see PR #41. a = 0, b = 1 - it's packet 0, of a total of 1.

Encoding happens [here](https://github.com/DakotaNelson/sneaky-creeper/blob/c7a4837c4241f774ee9ce302f1b42a1d25f8dee6/screep#L171) and decoding happens [here](https://github.com/DakotaNelson/sneaky-creeper/blob/c7a4837c4241f774ee9ce302f1b42a1d25f8dee6/screep#L101).

**Comment by ManickYoj** [comment]
Are we packetizing everything? Not just tweets? It seems this should be a modular part of encoders as they need it, since they have different packetization requirements and constraints. 

Also, I believe a README update is in order, since it's easier than reading the (admittedly wonderful) comments in the code. But seriously, nice comments :+1:

File: screep (hunk 1 above)
Comment (DakotaNelson) [inline screep:42]: I like to live on the edge.

**Comment by DakotaNelson** [comment]
So, everything is packetized and depacketized (words?) in the send/receive methods and then passed to the en/decoders as whole segments. The encoders shouldn't have to care about packetization at all - it should be completely transparent.

I hadn't put it in the README, no. My line of thinking is that it's happening in the background, so the user didn't need to know. That said, we could definitely use a "how this thing works" section in the readme.

**Comment by ManickYoj** [comment]
Ah - clarification - put the -v flag in the README. But first, ideally, fix it up so that it doesn't give output when the -v flag is absent. That is the intent, right?

**Comment by DakotaNelson** [comment]
Ohhhhhhhhhh, yeah. Good catch.

So, right now that output only happens in the `file` module, and it's because out output in the module itself.

**Comment by DakotaNelson** [comment]
Problem solved.

Eventually, the output functions should be moved to a `utils` file so that the modules can import them. That'd be nice. But for now, it works!

**Comment by ManickYoj** [comment]
Alright. I looked over the code and ran a couple tests and it seems to behave as intended (though I'm still not hot on the `a b` prepending). I changed one thing: the progress bar now adds +1 to every submitted n input so that it ends as `n of n` instead of `n-1 of n`. 

The error and notifications look great: nice job with the colors. I didn't test anything that should be packetized, so I can't speak for the progress bar when it's actually in use.

**Comment by DakotaNelson** [comment]
I've tested the progress bar in use and it worked great for me. YMMV. :) (Specifically, I only tested it on Linux. No idea how it works on other platforms. Honestly, no idea how any of this works on other platforms.)

What's your concern with the packet wrapping? (the `a b` prepending)

**Comment by ManickYoj** [comment]
Oh, with regards to the packet wrapping, it would just be nice if sending things to file or similar didn't output the packet count, since it's really not part of the end message.

**Comment by DakotaNelson** [comment]
Yeah, that's a fair point. The assumption I was making is that it'll be read in by "us" (sneaky-creeper) on the other end as well, and that the file wouldn't ever really be read "as is" by a person, manually.

**Comment by ManickYoj** [comment]
Ah. I suppose that's valid. I figured that on the computer to which you were dumping data, piping to file in that way was kind of nice. But there are other utilities for that, so I'm alright with it. Merging.

**Comment by DakotaNelson** [comment]
Cool. Yeah, I figured you'd do a `./screep receive -c file etc. etc. > somefile` to have it read from the "screep format" file and output to a "normal format" file. (Since you'd presumable have done some encoding/encryption/whatever that you'd need to undo anyway.)

**Pull request merged**